sdk/core-rest/core-client/review/core-client.api.md

```diff
@@ -0,0 +1,75 @@
+## API Report File for "@azure-rest/core-client"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { KeyCredential } from '@azure/core-auth';
+import { Pipeline } from '@azure/core-rest-pipeline';
+import { PipelineOptions } from '@azure/core-rest-pipeline';
+import { PipelineRequest } from '@azure/core-rest-pipeline';
+import { RawHttpHeaders } from '@azure/core-rest-pipeline';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export interface Client {
+    path: unknown;
```

Comment:
We know `path` will be a function at least, but probably no reason for this type to be pedantic.

Reply:
made it `Function`